src/proposals/20240320/SparkEthereum_20240320.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { ICapAutomator } from '../../interfaces/ICapAutomator.sol';
+import { IOwnable }      from '../../interfaces/IOwnable.sol';
+
+import { SparkPayloadEthereum } from '../../SparkPayloadEthereum.sol';
+
+/**
+ * @title  March 20, 2024 Spark Ethereum Proposal - Raise maximum supply cap in Cap Automator to 6,000 for WBTC and adjust DAI spread to 14% target APY
+ * @author Phoenix Labs
+ * @dev    This proposal changes the max parameter in supplyCapConfig in CapAutomator for WBTC market
+ * Forum:  https://forum.makerdao.com/t/mar-6-2024-proposed-changes-to-sparklend-for-upcoming-spell/23791
+ *         https://forum.makerdao.com/t/stability-scope-parameter-changes-11-under-sta-article-3-3/23910
+ * Votes:  TODO
```

Comment:
Votes are up:

https://vote.makerdao.com/polling/QmVGDsvm
https://vote.makerdao.com/polling/QmYYoAMe

Forum post about the Morpho vault will be posted by BA soon. They can do changes without polling on this.